packages/edit-site/src/components/page-pages/index.js

```diff
@@ -257,11 +265,25 @@ export default function PagePages() {
 			{
 				header: __( 'Status' ),
 				id: 'status',
-				getValue: ( { item } ) =>
+				fieldType: ENUMERATION_TYPE,
+				// TODO: we need to clarify what `getValue` returns.
+				// Probably it should just return the raw data connection with the item,
+				// So in `author` probably return the `userId`..
+				getValue: ( { item } ) => item.status,
+				setValue: ( { value, item } ) => {
+					editEntityRecord( 'postType', 'page', item.id, {
+						status: value,
+					} );
+					saveEditedEntityRecord( 'postType', 'page', item.id );
+				},
+				render: ( { item } ) =>
 					STATUSES.find( ( { value } ) => value === item.status )
 						?.label ?? item.status,
 				type: ENUMERATION_TYPE,
 				elements: STATUSES,
+				editElements: STATUSES.filter( ( { value } ) =>
```

Comment:
What do we need `editElements` for? I'd think the user should be able to select any of the elements' values.

Reply:
>Some enumeration fields can have different available options between filtering and editing. An example is the status field where some statuses would require more actions beside updating a single value - they could require from the user to set more values(future status needs also a date input).

Reply:
My point is: I don't see `editElements` as a target API we need. Instead, I feel we should introduce the `edit` function for fields that cannot use the default edit.

We don't need to do it all at once, of course. We can have a `editable: false` or something like that for fields that don't allow inline editing yet.